src/App/mermaidData/syncApiDataIntoOfflineStorage/SyncApiDataIntoOfflineStorage.js

```diff
@@ -6,8 +6,8 @@ const SyncApiDataIntoOfflineStorage = class {
 
   _dexiePerUserDataInstance
 
-  _getTableNamesWhereSyncingWasRejectedListedPerProject =
-    function _getTableNamesWhereSyncingWasRejectedListedPerProject(pushResponse) {
+  #getTableNamesWhereSyncingWasRejectedListedPerProjectFromSyncPushError =
```

Comment:
since pull will be different... most likely. If its not I will be reusing and renaming this, but for the time being its likely to stay different  enough from pull to be explicit about it with naming to help people out a little sans typescript.

Reply:
i'm confused by the # sign in front - are we using this convention instead of _ ?